editions/tw5.com/tiddlers/system/tw5.com-styles.tid

```diff
@@ -325,3 +325,22 @@ type: text/vnd.tiddlywiki
 .tc-btn-download:active {
 	box-shadow: none;
 }
+
+/* WikiText rules */
+
+.multi-columns,
```

Comment:
It will be used in the other PRs because it changes from 4 to 2 columns for small screens it's named multi-column. May be in the future we may change it to 3 columns if the tiddler names get longer. 

There are some more long lists where this class should be useful.

Reply:
OK thanks @pmario